### DISCUSSION TITLE
[Completed] Support of back and forward mouse buttons

### DISCUSSION THREAD

**Discussion by Herr-Sepp** [JPTGamesAndApps/ShresthaFiles#5] [Ideas]
I have a mouse with back and forward button.
In Windows Exporer, I can go back and forth in the folder tree with one click.

Unfortunately, this is not yet possible in ShresthaFiles. 
Both keys are recognized as normal left-click.

It would be great to have this feature also in ShresthaFiles.
Thanks for the great app

**Comment by JPTGamesAndApps** [comment]
Hello Herr-Sepp,

Recent Pro version of the app should support back and forward mouse button. The free version is usually behind the Pro version and may not have the feature yet.

Did you check for the update to see if you are using the latest version in the Microsoft Store?

Let me know if it still does not work!

**Reply by JPTGamesAndApps** [reply]
> 
> 
> I have a mouse with back and forward button.
> In Windows Exporer, I can go back and forth in the folder tree with one click.
> 
> Unfortunately, this is not yet possible in ShresthaFiles.
> Both keys are recognized as normal left-click.
> 
> It would be great to have this feature also in ShresthaFiles.
> Thanks for the great app

Version 0.43.x (being processed in the store) "should" finally resolve the navigation back/forward button issue.
Thank you for your patience!

**Reply by Herr-Sepp** [reply]
Thanks, i will check the version as soon as it is available for me (Full Licence)

**Reply by Herr-Sepp** [reply]
I'm using this a wile now and it is working great. Thanks.
You can close this Idea.

**Reply by JPTGamesAndApps** [reply]
Thanks for the update!
